.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
```

Comment:
Can you add clarification that pypi version will only be updated at most once per day in response to master branch changes?

Reply:
> Looks good, just two minor comments:
> 
> * Can you add clarification that pypi version will only be updated at most once per day in response to master branch changes?
> * Should the installation instructions on the website have an extra section discussing this?

Thanks @jwmueller I will add a comments:

```
# Note: the first change per day is used to build daily/night release.
# Therefore, merging one PR per day is recommanded. Otherwise, the we need to mannual trigger the CI later.
```